src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int side1;
+    private final int side2;
+
+    public Rectangle(String color,int side1, int side2) {
+        super(color);
+        this.side1 = side1;
+        this.side2 = side2;
+    }
+
+    @Override
+    public double area() {
+        return side1 * side2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle " + "area: " + area() + " sq. units"
```

Comment:
Use the name() method for getting String representation of enum constants instead of relying on the toString() method, to avoid issues if the toString() method is overridden.